.azure-pipelines/scripts/test_runner.sh

```diff
@@ -134,12 +138,24 @@ function GetReadyToRunNextTest()
 function SkipTestIfDisabled()
 {
     skip_test=false
+    # If this test is in $disabled_tests_file skip it
     is_test_disabled=$(grep -c "$file" "$disabled_tests_file")
-    # if this test is disabled set counters and skip to next test
     if [[ $is_test_disabled -ge 1 ]]; then
         echo "Test $file is disabled. Skipping test..."
         echo "To enable the test remove $file from $disabled_tests_file"
+    fi
 
+    # If this test is in $nightly_tests_file and this is not a nightly build skip it
+    if [[ $is_test_disabled -eq 0 && $SGXLKL_NIGHTLY_BUILD -eq 0 ]]; then
+        is_test_nightly_only=$(grep -c "$file" "$nightly_tests_file")
+        if [[ $is_test_nightly_only -ge 1 ]]; then
+            echo "Test $file is marked nightly build only. Skipping test..."
+            echo "To enable the test remove $file from $nightly_tests_file"
+        fi
+    fi
+
+    # if this test is disabled set counters and skip to next test
+    if [[ $is_test_disabled -ge 1 || $is_test_nightly_only -ge 1 ]]; then
```

Comment:
`$is_test_nightly_only -ge 1` is confusing. You can get rid of it by setting `$is_test_disabled` on line 152.

Reply:
Done. Got rid of is_test_nightly_only  and used is_test_disabled